frontend/app/src/components/accounts/blockchain/ChainDisplay.vue

```diff
@@ -1,25 +1,30 @@
 <script setup lang="ts">
 import { type Blockchain } from '@rotki/common/lib/blockchain';
 import ListItem from '@/components/helper/ListItem.vue';
+import { useSupportedChains } from '@/composables/info/chains';
```

Comment:
Import is not need. btw some times the auto-import might not update instantly so until it does the IDE suggests to import.